docs/release/2022q1.md

```diff
@@ -0,0 +1,142 @@
+## 2022-04-01: 2022q1 release
+
+This release covers everything from 2022-01-01 and has been tested with avr-gcc
+v11.2.0 from Upstream and arm-none-eabi-gcc 10.3-2021.10 from Arm.
+
+Breaking changes:
+
+- Removed all functions marked deprecated for >1 year.
+
+Features:
+
+- STM32L5 Support.
+- Raspberry Pi PR2040 Support.
+- Microchip SAM Timer driver.
+- Improved Project and Doxygen documentation.
```

Comment:
```suggestion
- STM32L5 support.
- Raspberry Pi PR2040 support.
- Microchip SAM timer driver.
- Improved project and Doxygen documentation.
```
Minor spelling nitpick, common nouns like "support" and "project" should not be capitalized here.